### DISCUSSION TITLE
Missing script in the deploy to azure series

### DISCUSSION THREAD

**Discussion by OluochOdhiambo** [skills/.github#215] [Q&A]
There seems to be a missing script that is blocking step2 (Set up an Azure Environment) from being cleared. Has anyone else encountered the issue?

![Screenshot 2023-04-05 160107](https://user-images.githubusercontent.com/63576010/230088651-027e5800-6e59-44f3-813a-08e4114e9ae9.png)

**Comment by SimonVerhoeven** [comment]
You can add the file yourself to pull main, there are also some other quirks in it like having to adapt the workflow to use node v16.
